### PR TITLE
Organization enable chat

### PR DESCRIPTION
**Pull request by reiterl**
Add field to model, generate models.py,
add field to orga.update 
add check to EnableChatMixin and update tests.

### CODE REVIEW

File: tests/system/action/chat_group/test_create.py [hunk 1]
```diff
@@ -5,7 +5,10 @@
 class ChatGroupCreate(BaseActionTestCase):
```
Comment (jsangmeister) [inline tests/system/action/chat_group/test_create.py:5]: Add a test where `meeting/enable_chat` is true, but `organization/enable_chat` is false and assert it fails

File: tests/system/action/chat_group/test_sort.py [hunk 2]
```diff
@@ -6,6 +6,7 @@ class ChatGroupSortActionTest(BaseActionTestCase):
     def setUp(self) -> None:
```
Comment (jsangmeister) [inline tests/system/action/chat_group/test_sort.py:6]: same here

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: tests/system/action/chat_group/test_create.py (hunk 1 above)
Comment (reiterl) [reply tests/system/action/chat_group/test_create.py:5]: Done.

File: tests/system/action/chat_group/test_sort.py (hunk 2 above)
Comment (reiterl) [reply tests/system/action/chat_group/test_sort.py:6]: Done.

**Comment by reiterl** [comment]
I also did a rebase to solve a conflict.

**Comment by reiterl** [comment]
Added the two missing tests. I think this can be merged.

**Pull request merged**